cli/src/builders/environment.ts

```diff
@@ -0,0 +1,38 @@
+
+export function getBranchLibraryName(currentBranch: string) {
+  const parts = branchSplit(currentBranch);
+  if (parts.length > 1) {
+    // We make A LOT of assumptions here about a valid branch name
```

Comment:
maybe document the assumed format?

Reply:
We just assume that devs use `/` and `-` in their branch names.